maven-artifact/src/main/java/org/apache/maven/artifact/versioning/ComparableVersion.java

```diff
@@ -150,6 +150,30 @@ public int compareTo( Item item )
             }
         }
 
+        @Override
+        public boolean equals( Object o )
```

Comment:
Why not use `Objects#equals()` on this?

Reply:
Because I wasn't sure about the minimum Java version of Maven.